src/models/models.py

```diff
@@ -56,12 +58,39 @@ def __repr__(self) -> str:
 class AccessToken(Base):
     __tablename__ = 'accesstokens'
 
-    id = mapped_column(Integer, primary_key=True, autoincrement=True, nullable=False)
-    registered_to_user = mapped_column(ForeignKey('users.id'), ForeignKey('users.id'), nullable=False)
-    created_at = mapped_column(DateTime, server_default=func.now(), nullable=False)
-    expires = mapped_column(DateTime, nullable=False)
-    token_value = mapped_column(String, nullable=False)
+    id: Mapped[Integer] = mapped_column(Integer, primary_key=True, autoincrement=True, nullable=False)
+    registered_to_user: Mapped[Integer] = mapped_column(ForeignKey('users.id'), ForeignKey('users.id'), nullable=False)
+    created_at: Mapped[DateTime] = mapped_column(DateTime, server_default=func.now(), nullable=False)
+    expires: Mapped[DateTime] = mapped_column(DateTime, nullable=False)
+    token_value: Mapped[String] = mapped_column(String, nullable=False)
 
     @validates('id', 'registered_to_user ', 'created_at', 'expires', 'token_value')
     def empty_string_to_null(self, key, value):
         return None if value == '' else value
+
+class Group(Base):
+    __tablename__ = 'groups'
+
+    id: Mapped[Integer] = mapped_column(Integer, primary_key=True, autoincrement=True, nullable=False)
+    name: Mapped[String] = mapped_column(String, nullable=False)
+    description: Mapped[String] = mapped_column(String, nullable=False)
+    created_at: Mapped[DateTime] = mapped_column(DateTime, server_default=func.now(), nullable=False)
+    members: Mapped[List["GroupMember"]] = relationship(back_populates="group", cascade='all, delete')
+
+    @validates('id', 'name ', 'description', 'created_at', 'members')
+    def empty_string_to_null(self, key, value):
+        return None if value == '' else value
+
+class GroupMember(Base):
+    __tablename__ = 'groupmembers'
+
+    id: Mapped[int] = mapped_column(Integer, primary_key=True, autoincrement=True, nullable=False)
```

Comment:
Why is this `Mapped[int]` but on line 74 it's `Mapped[Integer]`? Is this just type hinting for Python?

Reply:
Just me being lazy/inconsistent I guess. I'll switch both to Integer since that's the one that comes with SQL Alchemy